### DISCUSSION TITLE
Rules disabled and how it come back to work?

### DISCUSSION THREAD

**Discussion by bakerLecter** [jertel/elastalert2#1178] [Q&A] [answered]
Sometimes my ES have little change on index, and the rules will get some error pop out in std and auto-disabled by elastalert. Will it come back to enable after sometimes? Or just disables util I reboot the elastalert?

**Comment by jertel** [comment, marked as answer]
You can configure ElastAlert2 to stop disabling rules on errors:

![image](https://github.com/jertel/elastalert2/assets/16639611/d21e4f31-eb73-40e1-958d-9b6be8c7fdc1)

**Reply by bakerLecter** [reply]
That's great. But I wonder if that too many error, like I got at 20 to 30 error rules in some times when the es got some problems, could it cause elastalert crash?
